jerry-util/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required (VERSION 2.8.12)
+set(JERRY_UTIL_NAME jerry-util)
+project (${JERRY_UTIL_NAME} C)
+
+# Include directories
+set(INCLUDE_UTIL "${CMAKE_CURRENT_SOURCE_DIR}/arg-validator")
+
+# Source directories
+file(GLOB SOURCE_UTIL arg-validator/*.c)
+
```

Comment:
To keep it more modular: I would add another `jerry-util/arg-validator/CMakeLists.txt` and just recurse into it from here. I'd also create a new target for each util.